plugins/qtAccessionPlugin/lib/model/QubitDeaccession.php

```diff
@@ -45,8 +45,6 @@ public function save($connection = null)
 
   public function delete($connection = null)
   {
-    QubitSearch::getInstance()->delete($this->accession);
```

Comment:
If we are updating the index when we save/add an accession, then will we need to also '->update' on a delete? Or is the '->update' in save() not necessary either?

Reply:
I did some testing and looked at the code and it seems like the parent accession's document exists independently of whether or not there's a deaccession so I've removed both of the ElasticSearch alterations.

Reply:
(I've gotten rid of the `update` in the save because of this.)